### PR TITLE
Add support for Python 3.6, 3.7, 3.8 and 3.9.

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.

### CODE REVIEW

File: src/zc/intid/tests/test_subscribers.py [hunk 1]
```diff
@@ -118,13 +119,13 @@ def setUp(self):
 
         self.root['folder1'] = Folder()
         self.root['folder1']['folder1_1'] = self.folder1_1 = Folder()
-        self.root['folder1']['folder1_1']['folder1_1_1'] = self.child_folder = Folder()
+        self.root['folder1']['folder1_1']['folder1_1_1'] = Folder()
+        self.child_folder = self.root['folder1']['folder1_1']['folder1_1_1']
```
Comment (mgedmin) [inline src/zc/intid/tests/test_subscribers.py]:
```suggestion
        self.folder_1_1['folder1_1_1'] = self.child_folder = Folder()
```
?

File: src/zc/intid/tests/test_utility.py [hunk 2]
```diff
@@ -159,7 +158,7 @@ def test_btree_long(self):
         # This is a somewhat arkward test, that *simulates* the border case
```
Comment (mgedmin) [inline src/zc/intid/tests/test_utility.py]:
```suggestion
        # This is a somewhat awkward test, that *simulates* the border case
```

File: src/zc/intid/tests/test_zope_zcml.py [hunk 3]
```diff
@@ -79,13 +80,7 @@ class I(zope.interface.Interface):
...
-
 def test_suite():
     return unittest.TestSuite([
         unittest.makeSuite(TestZopeIntidZcml),
     ])
-
-test_suite() # coverage
-
-if __name__ == '__main__': # pragma: no cover
-    unittest.main()
```
Comment (mgedmin) [inline src/zc/intid/tests/test_zope_zcml.py:91]: I vote for removing these bits from the other test files too.

**Review by mgedmin — APPROVED** [review]
lgtm

File: src/zc/intid/tests/test_zope_zcml.py (hunk 3 above)
Comment (icemac) [reply src/zc/intid/tests/test_zope_zcml.py:91]: Okay.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**